cpp/include/raft/sparse/cusparse_wrappers.h

```diff
@@ -26,6 +26,24 @@
   case err:                       \
     return #err;
 
+//Notes:
+//(1.) CUDA_VER_SELECT aggregates all the CUDA version selection logic;
+//(2.) to enforce a lower version,
+//
+//`#define CUDA_ENFORCE_LOWER
+// #include <raft/sparse/cusparse_wrappers.h>`
+//
+// (i.e., before including this header)
+//
+
+#define CUDA_VER_SELECT (CUDART_VERSION >= 10100)
+
+#define VALUE_TO_STRING(x) #x
```

Comment:
I would make these names more obscure like` `_VALUE` or `_RAFT_PREPROC_VALUE`, since this header will be included in customer codebases, we could easily collide with another use of VALUE in another context.

Reply:
I agree. It's residual code, meaning I checked it in to have it somewhere in a branch in case I need it again for debugging. Not meant to make it in the release. I will remove all of that.

Reply:
Any objections on removing these "*VALUE*" macros?

Reply:
I'm ok with removing or renaming - either works. Maybe removing is even better

Reply:
Done (removed).